OrbitGl/FunctionsDataView.cpp

```diff
@@ -33,14 +33,34 @@ const std::vector<DataView::Column>& FunctionsDataView::GetColumns() {
   return columns;
 }
 
+bool FunctionsDataView::ShouldShowSelectedFunctionIcon(const FunctionInfo& function) {
+  return GOrbitApp->IsFunctionSelected(function);
+}
+
+bool FunctionsDataView::ShouldShowFrameTrackIcon(const FunctionInfo& function) {
+  if (GOrbitApp->IsFrameTrackEnabled(function)) {
```

Comment:
Can't we ensure, that this will return "true" whenever a frame tack for that function is drawn (disregarding the capture data - is connected case)?

The idea would be: DataManager stores all the FrameTrack Functions that are drawn or will be drawn in the next capture.
Use CaptureData's FrameTracks is just for serialization, but to do so, store all the functions of FrameTracks currently drawn.

On Deserialization, ensure, that we also enable the FrameTracks on the DataManager, but do not "enable"/"select" the functions, if that capture is not connected.

Reply:
It seems DataManager is not touched at all when deserializing data. Is it a good idea to now store some of the capture related data in DataManager?

The issue is that you can have frame tracks in the capture data but they will not be drawn when you capture again (as you can just unhook the function).

Reply:
We at least need to think carefully about this.
Right now, I do not see an issue of storing this information in data manager.

I see the issue you describe, that feels a bit strange. But maybe there is a solution also for that.

Unfortunately, need to go now. We can discuss this offline tomorrow in more detail.